modules/developer-preview/pages/preview-mode.adoc

```diff
@@ -0,0 +1,87 @@
+= Developer Preview Mode and Features
+
+[abstract]
+The Developer Preview mode, also known as Preview mode, provides early access to upcoming features and enables you to play with these features and provide feedback during the development phase. Note that these features are not supported by Couchbase Legal Agreements, may not be functionally complete, and are not intended for production use. They are intended for development and testing purposes only. 
+
+Introduced in Couchbase Server Enterprise Edition 6.5 Beta, the Developer Preview mode is enabled by default during the Beta release. When the Preview mode is enabled, the cluster is converted to a Developer Preview cluster and cannot be switched back. Given that the Developer Preview features are under development, Couchbase cannot guarantee the stability of these features. As a result, you cannot upgrade to subsequent releases.
+
+While the Developer Preview Mode can be enabled using the CLI or REST API, once enabled, it cannot be disabled. 
+
+Note that the commands on this page assume a Ubuntu Linux environment. For the appropriate command-paths, see xref:cli:cli-intro.adoc[CLI Reference].
```

Comment:
See previous comment. Would prefer "an Ubuntu" but be consistent with usage in the rest of the docs.